x-pack/plugins/endpoint/common/generate_data.ts

```diff
@@ -353,8 +474,11 @@ export class EndpointDocGenerator {
       maxChildrenPerNode,
       relatedEventsPerNode,
       percentNodesWithRelated,
-      percentChildrenTerminated
+      percentChildrenTerminated,
+      alwaysGenMaxChildrenPerNode
     );
+    // return the origin/alert event
+    return ancestry[ancestry.length - 1];
```

Comment:
what is this used for? this will be a duplicate of an event that is already returned on line 468

re-commenting because my first comment was on a specific commit, not the PR

Reply:
Oh thanks for pointing that out. I need to remove that. I was going to use that as a way to get the origin of the tree, but I don't need to anymore now that I created the `generateTree` method.